Twitter Profile Page/js/editProfile.js

```diff
@@ -0,0 +1,89 @@
+var PromiseList=[ "https://fsd1.herokuapp.com/users/4/details",
+				];
+let  requests = PromiseList.map(url => fetch(url));
+let store=[];
+	Promise.all(requests)
+			.then(responses=>{
+				responses.forEach(response=>store.push(response.json()))
+			})
+			.catch(error=>console.log(error));
+			
+async function PromiseCall(url){
+	let response= await fetch(url);
+	let data=await response.json();
+	return data;
+}
+
+async function response(url,init){
+	let PromiseCall=await fetch(url,init);
+	let data=await PromiseCall.json()
```

Comment:
Do check to see if there is an error. Don't always just call .json()

Reply:
If i remove .json call i am getting data undefined error